apps/www/components/Pricing/ComputePricingTable.tsx

```diff
@@ -28,20 +28,7 @@ const ComputePricingTable = () => {
           <tbody>
             {pricingAddOn.database.rows.map((row, i) => (
               <Fragment key={`row-${i}`}>
-                {i === 0 && (
-                  <tr>
-                    <td className="pb-1 bg-border-strong px-3 py-1 -mr-1 border-l-4 border-strong">
-                      <span>First instance is free on paid plans</span>
-                    </td>
-                  </tr>
-                )}
-                <tr
-                  key={i}
-                  className={cn(
-                    i % 2 === 0 ? 'bg-surface-100 rounded-lg' : '',
-                    i === 0 ? 'border-4 border-strong' : ''
-                  )}
-                >
```

Comment:
Took this out b/c it looked weird with the introduce of the Nano instance in the row above. Also seemed redundant since there's mention of $10 of compute credits which is applied to make the first Micro instance free.

Here's what it would look like if it's kept in:

<img width="1257" alt="Screenshot 2024-04-06 at 17 54 45" src="https://github.com/supabase/supabase/assets/5532241/d3d95a0d-ca19-47c3-bf89-c079db812b39">